### ISSUE TITLE
URL anchors are detected as invalid hex colors

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#387]
In the following CSS: (I've truncated the base64 hex string for readability)

```css
@font-face {
	font-family: dashicons;
	src: url(data:application/font-woff;charset=utf-8;base64, ABCDEF==) format("woff"),
		url(../fonts/dashicons.ttf) format("truetype"),
		url(../fonts/dashicons.svg#dashicons) format("svg");
	font-weight: normal;
	font-style: normal;
}
```

I get a warning: 
```
10:29	⚠  Unexpected invalid hex color "#dashicons" (color-no-invalid-hex) [stylelint]
```

**Comment by davidtheclark** [comment]
I added your example CSS as a test, and a check for that rule that ensures the `#` is only attended to if it follows a colon, comma, or whitespace. I'm not 100% sure that that will catch only hex values without errors, but I think it's a step in the right direction.

**Comment by ntwb** [comment]
Agreed, in the right direction for sure :+1: 

Confirming as fixed for my outlined use case :+1:
